### DISCUSSION TITLE
2 N00b questions on labs for hero 12

### DISCUSSION THREAD

**Discussion by EmperorKabir** [gopro/labs#601] [Q&A]
Just getting the 12 in a few days. If I generate a qr code with the bit rate set to 200 continuous and various other 'standard' settings and apply it to labs firmware by scanning the qr code, but then while shooting I decide to change a setting on the camera itself (if this is possible with labs firmware?) Eg I adjust EV compensation, Will the other 'hidden' settings be lost? Eg will the 200 bit rate go back to 'stock'? (Presumably this is not something I can confirm by looking at the settings on the camera itself).

And a slight extension question, is there a sort of onsetting/offsetting multiplier effect between settings? Eg is 200mb continuous with 'high' bit rate overall higher than 200mb continuous not on high or is the 200 mb continuous an absolute override and therefore the same in either case?

Thank you very much for your help!

**Comment by dnewman-gpsw** [comment]
The BITR and all other FourrCC extensions are global, and can only be enabled or disabled through QR codes.  There don't care that much about UI settings.  Some QR codes set things in the UI, like x-1 for EV -1, and i8 for ISO MAX 800, then the UI can change those back.

**Reply by EmperorKabir** [reply]
Thanks for that, any ideas on my second question?

**Reply by dnewman-gpsw** [reply]
Same answer, the controls are global, UI is not related.

**Reply by EmperorKabir** [reply]
I'm not sure I fully understand, especially as you can set bit rate as high/standard in something like qrcontrol as well as set BITR to 200 as part of the same qr code. I'm trying to figure out if one overrides the other or if they interact

**Reply by dnewman-gpsw** [reply]
Separate. BITR is global, like all FourCCs. b1 - Bit-rate high, just set the UI from the current preset (H11 or earlier), global UI on H12. They are unrelated.

Best to experiment, as this can change.

**Reply by EmperorKabir** [reply]
Ah ok so by being separate, they interact as independent multipliers and one does not override the other
